Makefile.d/functions.mk

```diff
@@ -89,3 +89,27 @@ define telepresence
 	    ## will be available after telepresence 0.105 released
 	    ## --deployment-type "$(SWAP_DEPLOYMENT_TYPE)"
 endef
+
+define gen-test
```

Comment:
resolve the following tickets with other PR.
https://github.com/vdaas/vald/projects/3#card-36918740

cc: @vankichi @kevindiu